docs/diagrams/DeleteSequenceDiagram.puml

```diff
@@ -49,12 +49,24 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
-DeleteCommand -> Model : deletePerson(1)
+DeleteCommand -> DeleteCommand : deleteWithIndex(m)
+activate DeleteCommand
+
+DeleteCommand -> Model : deletePerson(p)
+activate Model
+
+Model -> Model : updateFilteredPersonList(predicate)
```

Comment:
I am not sure, need to revise the notes but do we need to show the internals of the model?